### PR TITLE
Rewire views/UI and update readme

### PR DESCRIPTION
**Pull request by tmikeschu**
Making a WIP to stay on top of CI builds and such. More to come!

### CODE REVIEW

File: src/App/Map/PathMapContainer/PathMapContainer.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    expect(mapContainer).toBeTruthy();
+  });
+
+  it('renders a single PathMap child', () => {
+    const mapContainer = shallow(<PathMapContainer />);
+    expect(mapContainer.children().length).toEqual(1);
+    expect(mapContainer.children()['node']['type']).toEqual(PathMap)
+  });
+});
+
```
Comment (DavidKnott) [inline src/App/Map/PathMapContainer/PathMapContainer.test.js]: Great job figuring out how to test the map @tmikeschu !!!!

**Comment by DavidKnott** [comment]
Great work, love how you extracted the map!

**Comment by tmikeschu** [comment]
screen shot 2017-04-05 at 6 21 34 am

**Comment by tmikeschu** [comment]
@DavidKnott @ethanbennett @ski-climb @case-eee ready for review!

I'm having problems with end-to-end testing. The google maps script tag has an async load time, and when I try to `mount` any component with Enzyme, I get a `google is not defined` error.

Update map tests
Remove pathway components from App render
Add tests for PathMap and PathMapContainer
Add tests and shell for ProductSearch component
Update App unit tests for rendering ProductSearch
Update tests, views, and add styling
Add .contains pattern in unit tests
Setup new root view with styling
Add polyline functionality
Add product search item componenet and styling
Add click listener for product search item
Add listener for view all paths
Add functionality for searching for a product
Message for no results
Enter key submits query
View All button is conditional on results

[Finishes #142978585]

**Comment by DavidKnott** [comment]
@tmikeschu I like how you stubbed the information you'll be getting, it should simplify integrating the client with our backends.

**Comment by tmikeschu** [comment]
@DavidKnott that's the hope!

File: README.md [hunk 2]
```diff
@@ -24,9 +24,10 @@ change of possession (e.g., between producer and transport).
 
 ### About Us
 
-We are a group of four [Turing School]() students, and this is our capstone
+We are a group of four [Turing School](https://www.turing.io/) students, and this is our capstone
 project for the back-end engineering program. You can learn more about our
-development process at these links: [DTR (team expectations)](), [stand-ups](), [Pivotal Tracker]().
+development process at these links: [DTR (team expectations)](https://gist.github.com/tmikeschu/1a37eef7724f06421d8bca19fd8be8ad), [stand-ups](https://gist.github.com/tmikeschu/14555f11f0d24d7c09749519c7b04337), [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/1996565).
+And more about us at these ones: [Nick](https://github.com/ski-climb), [David](https://github.com/DavidKnott), [Ethan](https://github.com/ethanbennett), [Mike](https://github.com/tmikeschu).
```
Comment (ethanbennett) [inline README.md]: Thanks for updating the links!

**Comment by tmikeschu** [comment]
@ethanbennett easy with the M word there!

**Pull request merged**